### PR TITLE
Hg 829 Adding comments and removing unused gulp task

### PR DESCRIPTION
**Pull request by lizlux**
I started this as part of https://wikia-inc.atlassian.net/browse/HG-829 but now it's just adding comments and code cleanup. 

@katanka @kenkouot

### CODE REVIEW

**Comment by katanka** [comment]
LGTM

File: gulp/tasks/build-views.js [hunk 1]
```diff
@@ -57,8 +61,9 @@ gulp.task('build-views', ['scripts-front', 'copy-ts-source', 'vendor', 'build-ve
 		)),
 
 		gulpif('**/_layouts/*.hbs', gulp.dest('www/server/views/_layouts')),
+
+		// for ember-main.hbs, find the file paths in the template source and replace them for prod CDN
 		gulpif(environment.isProduction, piper(
```
Comment (rogatty) [inline gulp/tasks/build-views.js]: How does it know to do this only for `ember-main.hbs`?

Comment (lizlux) [inline gulp/tasks/build-views.js]: https://github.com/Wikia/mercury/blob/dev/gulp/options/prod.js#L53

This file could use some more cleanup for sure, but this PR is a start.

Comment (rogatty) [inline gulp/tasks/build-views.js]: Oh, right, thanks! That's the info that we've missed when quickly investigating with @bkoval.

**Pull request merged**